### PR TITLE
main/gdb: upgrade to 8.3

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: main/gdb/APKBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gdb
-pkgver=8.2.1
-pkgrel=2
+pkgver=8.3
+pkgrel=0
 pkgdesc="The GNU Debugger"
 url="https://www.sourceware.org/gdb/"
 arch="all"
```
Comment (maxice8) [inline main/gdb/APKBUILD]: empty variables can be removed for brevity

Comment (maxice8) [reply main/gdb/APKBUILD]: GPL-3.0 is not a valid SPDX identifier

File: main/gdb/APKBUILD [hunk 2]
```diff
@@ -15,7 +15,6 @@ source="https://ftp.gnu.org/gnu/$pkgname/$pkgname-$pkgver.tar.xz
 	s390x-use-elf-gdb_fpregset_t.patch
 	ppc-musl.patch
 	ppc-ptregs.patch
-	aarch64-headers.patch
 	"
 
 builddir="$srcdir"/$pkgname-$pkgver
```
Comment (maxice8) [inline main/gdb/APKBUILD]: default value for builddir, can be removed.

File: main/gdb/APKBUILD (hunk 1 above)
Comment (J0WI) [reply main/gdb/APKBUILD]: Changed to `GPL-3.0-or-later LGPL-3.0-or-later`

**Comment by J0WI** [comment]
Fixed in ef7c6ebe6a056a2777405f4c0749d318eda28e6b

**Pull request closed without merging**